sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Utils.java

```diff
@@ -646,4 +648,11 @@ public static String toJson(ObjectMapper mapper, ObjectNode object) {
             throw new IllegalStateException("Unable to convert JSON to STRING", e);
         }
     }
+
+    public static byte[] serializeObjectToByteArray(Object obj) throws IOException {
+        ByteArrayOutputStream out = new ByteArrayOutputStream();
+        ObjectOutputStream os = new ObjectOutputStream(out);
+        os.writeObject(obj);
+        return out.toByteArray();
```

Comment:
Should be close the stream  ?

Reply:
The objects should get destroyed once out of the function